### DISCUSSION TITLE
Local DNS with Adguard on vServer

### DISCUSSION THREAD

**Discussion by Cilenco** [AdguardTeam/AdGuardHome#5419] [Q&A]
I deployed Adguard on my vServer in the internet. Everything works fine except DNS entries for my local devices like my FritzBox.

Can I get that working even tho my Adguard is not in my local network? In my local network I would know how to do it but with it on my vServer I'm not even sure if this is possible.

Any help is appreciated :)

**Comment by Nordlicht-13** [comment]
I also want to know how to configure the FritzBox router with a Adguard-Home running on a public vServer with a public IP?
I set up Adguard following this guide: https://adguard.com/en/blog/adguard-home-on-public-server.html#configuringclients

Do I have to enter the IP here in the Preverred and Alternativ DNS-Server?
<img width="787" height="599" alt="image" src="https://github.com/user-attachments/assets/2deee080-8a66-4165-aeb9-169b4a04bb96" 
/>

Or do I have to enter it under Local DNS-Server?
<img width="1031" height="818" alt="image" src="https://github.com/user-attachments/assets/6cb54726-a1ac-4ce5-ba6e-8f866782b4b1" />
